### PR TITLE
admin pages to bs4

### PR DESCRIPTION
**Pull request by kristinmerbach**
# PR Checklist

Please check if your PR fulfills the following requirements
- [x] The title follows our [guidelines](https://github.com/ideacrew/enroll/blob/trunk/CONTRIBUTING.md#commit)
- [ ] Tests for the changes have been added (for bugfixes/features), they use let helpers and before blocks
- [ ] For all UI changes, there is cucumber coverage
- [x] Any endpoint touched in the PR has an appropriate Pundit policy. For open endpoints, reasoning is documented in PR and code
- [x] Any endpoint modified in the PR only responds to the expected MIME types.
- [x] For all scripts or rake tasks, how to run it is documented on both the PR and in the code
- [x] There are no inline styles added
- [x] There are no inline javascript added
- [x] There is no hard coded text added/updated in helpers/views/Javascript. New/updated translation strings do not include markup/styles, unless there is supporting documentation
- [x] Code does not use .html_safe
- [x] All images added/updated have alt text
- [x] Doesn’t bypass rubocop rules in any way

# PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature (requires Feature flag)
- [ ] Data fix or migration (inert code, no impact until run)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Dependency updates (e.g., add a new gem or update to a version)

# What is the ticket # detailing the issue?

Ticket: [bootstrap admin pages](https://www.pivotaltracker.com/story/show/188304768)
https://www.pivotaltracker.com/story/show/188318939

# A brief description of the changes

Current behavior: admin pages were not in the new bs4 format

New behavior: admin pages are started in the new bs4 format

# Feature Flag

For all new feature development, a feature flag is required to control the exposure of the feature to our end users. A feature flag needs a corresponding environment variable to initialize the state of the flag. Please share the name of the environment variable below that would enable/disable the feature and which client(s) it applies to.

Variable name: BS4_ADMIN_FLOW_IS_ENABLED

- [ ] DC
- [x] ME

# Additional Context
Include any additional context that may be relevant to the peer review process.

### CODE REVIEW

**Comment by TristanB17** [comment]
@kristinmerbach Would we want to consider adding a `:bs4_admin_flow` flag to these changes?

File: app/controllers/exchanges/broker_applicants_controller.rb [hunk 1]
```diff
@@ -137,4 +136,12 @@ def find_broker_applicant
   def check_hbx_staff_role
     redirect_to exchanges_hbx_profiles_root_path, :flash => { :error => "You must be an HBX staff member" } unless current_user.has_hbx_staff_role?
   end
+
+  def enable_bs4_layout
+    @bs4 = true
+  end
+
+  def resolve_layout
+    EnrollRegistry.feature_enabled?(:bs4_admin_flow) ? "progress" : "single_column"
```
Comment (charlienparker) [inline app/controllers/exchanges/broker_applicants_controller.rb:145]: is progress right here, or can we use `bootstrap_4` instead?

**Review by charlienparker — DISMISSED** [review]

File: app/controllers/exchanges/broker_applicants_controller.rb (hunk 1 above)
Comment (kristinmerbach) [reply app/controllers/exchanges/broker_applicants_controller.rb:145]: I'm trying to standardize everything to progress for consistency as progress can work for pages with or without a side nav

**Review by TristanB17 — DISMISSED** [review]
lgtm

**Review by TristanB17 — APPROVED** [review]

**Review by charlienparker — APPROVED** [review]

**Pull request merged**